guides/common/modules/proc_installing-amazon-ec2-plugin.adoc

```diff
@@ -7,10 +7,12 @@ Install the Amazon EC2 plugin to attach an EC2 compute resource provider to {Pro
 This allows you to manage and deploy hosts to EC2.
 
 .Procedure
-. Install the EC2 compute resource provider on your {ProjectServer}:
+* Install the EC2 compute resource provider on your {ProjectServer}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # {foreman-installer} --enable-foreman-compute-ec2
 ----
-. Optional: In the {ProjectWebUI}, navigate to *Administer* > *About* and select the *compute resources* tab to verify the installation of the Amazon EC2 plugin.
+
+.Verification
+* In the {ProjectWebUI}, navigate to *Administer* > *About* and select the *Available Providers* tab to verify the installation of the Amazon EC2 plugin.
```

Comment:
```suggestion
* In the {ProjectWebUI}, navigate to *Administer* > *About*.
* On the *Available Providers* tab, verify the installation of the Amazon EC2 plugin.
```

Reply:
```suggestion
. In the {ProjectWebUI}, navigate to *Administer* > *About*.
. On the *Available Providers* tab, verify the installation of the Amazon EC2 plugin.
```
In that case we have to use a numbered list.

Reply:
Only when it has a single step.